app/sidekiq/bgs/submit_form686c_job.rb

```diff
@@ -18,45 +18,52 @@ class Invalid686cClaim < StandardError; end
-        user ||= BGS::SubmitForm686cJob.generate_user_struct(vet_info)
-        CentralMail::SubmitCentralForm686cJob.perform_async(saved_claim_id,
-                                                            KmsEncrypted::Box.new.encrypt(vet_info.to_json),
-                                                            KmsEncrypted::Box.new.encrypt(user.to_h.to_json))
-      else
-        DependentsApplicationFailureMailer.build(user).deliver_now if user&.email.present? # rubocop:disable Style/IfInsideElse
-      end
+
+      BGS::SubmitForm686cJob.send_backup_submission(vet_info, saved_claim_id)
     end
 
     def perform(user_uuid, icn, saved_claim_id, encrypted_vet_info)
-      @vet_info = JSON.parse(KmsEncrypted::Box.new.decrypt(encrypted_vet_info))
       Rails.logger.info('BGS::SubmitForm686cJob running!', { user_uuid:, saved_claim_id:, icn: })
-
-      @user = BGS::SubmitForm686cJob.generate_user_struct(@vet_info)
-      @user_uuid = user_uuid
-      @saved_claim_id = saved_claim_id
+      instance_params(encrypted_vet_info, icn, user_uuid, saved_claim_id)
 
       in_progress_form = InProgressForm.find_by(form_id: FORM_ID, user_uuid:)
       @in_progress_copy = in_progress_form_copy(in_progress_form)
 
-      claim_data = normalize_names_and_addresses!(valid_claim_data)
-
-      BGS::Form686c.new(user, claim).submit(claim_data)
-
-      # If Form 686c job succeeds, then enqueue 674 job.
-      BGS::SubmitForm674Job.perform_async(user_uuid, icn, saved_claim_id, encrypted_vet_info, KmsEncrypted::Box.new.encrypt(user.to_h.to_json)) if claim.submittable_674? # rubocop:disable Layout/LineLength
+      submit_forms(encrypted_vet_info)
 
       send_confirmation_email
       in_progress_form&.destroy
       Rails.logger.info('BGS::SubmitForm686cJob succeeded!', { user_uuid:, saved_claim_id:, icn: })
     rescue => e
+      handle_filtered_errors!(e:, encrypted_vet_info:)
+
       Rails.logger.warn('BGS::SubmitForm686cJob received error, retrying...',
-                        { user_uuid:, saved_claim_id:, icn:, error: e.message })
+                        { user_uuid:, saved_claim_id:, icn:, error: e.message, nested_error: e.cause&.message })
       log_message_to_sentry(e, :warning, {}, { team: 'vfs-ebenefits' })
       salvage_save_in_progress_form(FORM_ID, user_uuid, @in_progress_copy) if @in_progress_copy.present?
       raise
     end
 
+    def handle_filtered_errors!(e:, encrypted_vet_info:)
+      filter = FILTERED_ERRORS.any? { |filtered| e.message.include?(filtered) || e.cause&.message&.include?(filtered) }
+      return unless filter
+
+      Rails.logger.warn('BGS::SubmitForm686cJob received error, skipping retries...',
+                        { user_uuid:, saved_claim_id:, icn:, error: e.message, nested_error: e.cause&.message })
+
+      vet_info = JSON.parse(KmsEncrypted::Box.new.decrypt(encrypted_vet_info))
+      self.class.send_backup_submission(vet_info, saved_claim_id)
```

Comment:
we need to salvage the in progress form here.
I dont think we ever use it but would be good to follow the same paradigm as other failures.

Reply:
see line 43.